frontend/src/scenes/session-recordings/player/inspector/v2/PlayerInspectorList.tsx

```diff
@@ -25,12 +24,7 @@ import { LemonSkeleton } from 'lib/components/LemonSkeleton'
 import { userLogic } from 'scenes/userLogic'
 import { PayGatePage } from 'lib/components/PayGatePage/PayGatePage'
 import { IconWindow } from '../../icons'
-
-const TabToIcon = {
-    [SessionRecordingPlayerTab.EVENTS]: <UnverifiedEvent />,
-    [SessionRecordingPlayerTab.CONSOLE]: <IconTerminal />,
-    [SessionRecordingPlayerTab.PERFORMANCE]: <IconGauge />,
-}
+import { tabToIconAndDescription } from 'scenes/session-recordings/player/inspector/v2/PlayerInspectorControls'
```

Comment:
Could we put this somewhere else rather than importing from each other? Also, given that they have two different rendering cases, this might the wrong case for reusability.